activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -894,6 +894,10 @@ def remove_index_for_alter(table_name, column_name = nil, **options)
           "DROP INDEX #{quote_column_name(index_name)}"
         end
 
+        def supports_values_syntax?
```

Comment:
```suggestion
        def supports_insert_values_syntax?
```

perhaps?

Reply:
pushed in https://github.com/rails/rails/pull/51348/commits/6a4fd544e75e0c8413b36de5e7c87ffb25d403bb